recipes/samplot/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "samplot" %}
+{% set version = "1.0.1" %}
+{% set sha256hash = "2e10ad811ac9909a31c846f625246d25e7482375245fd7c4835d5a9777cbfba2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ryanlayer/samplot/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256hash }}
+
+build:
+  number: 0
+  skip: True  # [osx]
```

Comment:
Is there a reason why OSX is skipped? All Python packages are available for OSX as well.

Reply:
Same as above, a remnant that can be removed. Thanks for pointing those out!